defaults/main.yml

```diff
@@ -4,6 +4,7 @@
 # Default is only rsa; ed25519 is recommended if supported by sshd.
 ssh_host_keys:
   - /etc/ssh/ssh_host_rsa_key
+  - /etc/ssh/ssh_host_ecdsa_key
```

Comment:
Why enable dsa keys? They were deprecated in OpenSSH 7.0? http://www.openssh.com/legacy.html

Reply:
Was reading it wrong. ECDSA should be okay. But anyway, why would you want to enable them?

Reply:
The switch is default on SLE15. It also seems to be default in openssh 7.5 and later.

Reply:
The main concern is that legacy OS Support could be broken. I.e. centos 6 or debian 7.

Reply:
We should make it depend on the distro version used then. At least CentOS 6 is still supported until November 30, 2020.
I do honestly not care about Debian 7 as it is EOL since May 31, 2018 and should be upgraded.